### DISCUSSION TITLE
MHIG Audi a3 Firmware k1555 problem fec mapa

### DISCUSSION THREAD

**Discussion by MEGI313** [Mr-MIBonk/M.I.B._More-Incredible-Bash#433] [Q&A]
Cześć mam problem z instalacją mapy 2024 posiadam audi a3 orginalny firmware MHIG_EU_AU_S1048 wczoraj wgrałem MHIG_EU_AU_K1555+ patch. Dodalem Fec 023000ee obecnie w radiu są 2 kody legalne : 02100016 i 023000ee . 
Niestety nie mogę wgrać mapy w czerwonym menu po wyborze aktualizacji i sprawdzeniu pliku z kardy sd opcja aktualizuj jest szara nie dostępna. Czy jest to wina orginalnego fec z starymi mapami? Czy może źle dodaje kody fec? Jak usunąć nie potrzebne kody?

**Comment by pemospec** [comment]
Udało się rozwiązać?
